MH-FiorellaTheme5/source/stylesheets/pdp-styles.css

```diff
@@ -0,0 +1,274 @@
+.product .product-title {
+  font-family: 'MerriweatherSans';
+  font-size: 28px;
+}
+
+.product .product-gallery-description {
+  margin-bottom: 20px;
+}
+.product .product-gallery-description b {
+  color: #555555;
+  font-family: 'MerriweatherSans-Bold';
+}
+
+.product .product-gallery-description p {
+  color: #555555;
+  display: inline;
+  font-family: 'MerriweatherSans-Regular';
+}
+
+.product .product-bouquet {
+  align-self: flex-start;
+  color: #555555;
+  font-family: 'MerriweatherSans-Regular';
+}
+
+.product .colors-bouquet {
+  padding: 20px;
+}
+
+.product .colors-bouquet span {
+  border-radius: 25px;
+  display: inline-block;
+  height: 50px;
+  width: 50px;
+}
+
+.product .colors-bouquet span:nth-child(1) {
```

Comment:
Hmmm not a big fan of specifying children. I would recommend to add a class to each and one of the `span` and add the styles from there.